docs/deployment_checklist.md

```diff
@@ -0,0 +1,47 @@
+# Deploying Apply - Post-deployment checks
+
+After deploying a new version of the service we have a short list of
+tests to run through. The aim is to detect any obvious issues that have
+been caused by the new release.
+
+This checklist is meant as a supplement to the full [deployment guide](deployment.md)
+where you will find detailed instructions about how to deploy.
+
+It may be appropriate carry out additional checks specific to the
+changes included in a deployment.
+
+We aim to work in pairs on deployments so that one developer runs the
+deployment itself and a second developer runs through this checklist.
+
+## 1. Production checks
+
+When the production deployment has finished ask another developer to go
+through the following checks:
+
+1. Notify the rest of the team via Slack that production is deployed and
+   that final checks are being made.
+2. Check
+   [Sentry](https://sentry.io/organizations/dfe-bat/issues/?project=1765973)
+   for runtime errors. These should also be shared in Slack in the
+   #twd_apply_tech channel.
+3. Create a new account on the system with a real email address and
+   check that you receive an email with the magic link and can use it to
+   log in. Use your DfE digital email address so that we can distinguish
+   test users.  If needed you can suffix your name with `+n` to create a
+   unique email address.
```

Comment:
I'm not sure we should do this on production. It will generate a lot of extra database and Slack noise.

Reply:
I was concerned about the database 'pollution', I asked about this on Friday and @benilovj reckons it's OK. I guess we can reuse the DfE email addresses - we just logout and get a new magic link email for an existing account rather than signing up again.

Reply:
I also think it's fine, since we have a way to filter them out of reporting. At some point within the next year, the number of legitimate candidates should also far outnumber anything we can create via manual testing. As long as we're not submitting applications, don't see much of a downside. 👍

Reply:
So following face-face discussion with @tijmenb and @benilovj I've changed the manual check for production to a simple 'open the home page to make sure it's running'.

Reply:
Hmm OK, as long as we do test the emails in QA and staging I think this works for now!